package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/anc95/jest-chain-transform#readme",
   "peerDependencies": {
-    "@jest/transform": "^27.0.6"
+    "@jest/transform": "^29.4.1"
```

Comment:
could you change this to be ">=27", so that it's Compatible with 27/28/29.. versions

Reply:
@jdufresne